tools/testdata-scripts/src/bin/make-testdata.rs

```diff
@@ -34,9 +14,9 @@ fn main() {
         .unwrap();
 
     let source_data = SourceData::default()
-        .with_cldr(paths::cldr_json_root(), CldrLocaleSubset::Full)
+        .with_cldr_for_tag(&icu_testdata::versions::cldr_tag(), CldrLocaleSubset::Full)
```

Comment:
Issue: Please, please make this read from the repo. A dev should be able to run `cargo make testdata` without an internet connection. This has always worked and there's no reason to break it.

Reply:
The repo might not reflect the correct data for this tag. This is more of an issue for the follow-up PR, where we generate testdata at `SourceData::LATEST_TESTED_CLDR_TAG` and then write that tag into the metadata. Local data could be wrong because someone forgot a glob, or we just haven't run the download script with the latest tag. I'm also more confident calling the constant `LATEST_TESTED_CLDR_TAG` if we actually test with it.

"Download" has been required for many things locally and on CI for quite a while now. Because it caches it's a one-time thing and we haven't had issues with it.

Reply:
The solution we've discussed is that there should be a CI action that verifies that the repo data is not stale.

> "Download" has been required for many things locally and on CI for quite a while now.

CI yes, but we've kept local development as no-network as possible.

It's also much easier to reason about the data pipeline if I can see exactly what data is being consumed by datagen in order to produce the processed ICU4X output data. Depending on some artifact downloaded from the network is simply not a good idea for the purposes of ICU4X development.

Reply:
I'd argue relying on the network is much less of a problem locally than in CI.  Then work-on-plane use case is rare, and even then you're gonna have the data in the cache unless you started ICU4X development just before boarding.

Reply:
Do you disagree with my point about calling the latest-tested-tag tested?